tests/integration-tests/configs/common/common.yaml

```diff
@@ -236,6 +236,12 @@ iam:
         schedulers: ["awsbatch", "slurm", "sge"]
         oss: ["alinux2"]
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
+  test_iam.py::test_s3_read_write_resource:
+    dimensions:
+      - regions: ["us-east-1"]
```

Comment:
How did you select us-east-1 here? I ask because in our last round of integration tests us-east-1 had the most tests run in it as compared to the other regions. Could these be moved to eu-central-1 instead?

Reply:
I randomly selected the region before. Done! It is moved to eu-central-1 now